roles/openshift_master/tasks/main.yml

```diff
@@ -24,7 +24,7 @@
   when: openshift_master_ha | bool and openshift_master_cluster_method == "pacemaker" and openshift.common.is_containerized | bool
 
 - name: Install Master package
-  action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-master{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
+  action: "{{ ansible_pkg_mgr }} name={{ openshift.common.service_type }}-master-{{ openshift_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }} state=present"
```

Comment:
@sdodson I'm wondering if it's the use of:
> `oo_image_tag_to_rpm_version={{ openshift_version }}`

There seems to be a lot going on here, it could also potentially be that depending on the version of ansible that `include_dash=True` is not being evaluated properly within the `oo_image_tag_to_rpm_version` filter, maybe we need to attempt to cast the variable rather than assuming it is a boolean value.